scala-test-suite/src/test/resources/2.13.4/BlacklistedTests.txt

```diff
@@ -0,0 +1,216 @@
+scala/tools/nsc/transform/ReleaseFenceTest.scala
+scala/tools/nsc/transform/SpecializationTest.scala
+scala/tools/nsc/transform/ThicketTransformerTest.scala
+scala/tools/nsc/transform/UncurryTest.scala
+scala/tools/nsc/transform/delambdafy/DelambdafyTest.scala
+scala/tools/nsc/transform/patmat/SolvingTest.scala
+scala/tools/nsc/transform/patmat/PatmatBytecodeTest.scala
+scala/tools/nsc/typechecker/ImplicitsTest.scala
+scala/tools/nsc/typechecker/InferencerTest.scala
+scala/tools/nsc/typechecker/NamerTest.scala
+scala/tools/nsc/typechecker/OverridingPairsTest.scala
+scala/tools/nsc/typechecker/ParamAliasTest.scala
+scala/tools/nsc/typechecker/TypedTreeTest.scala
+scala/tools/nsc/util/StackTraceTest.scala
+scala/util/ChainingOpsTest.scala
+
+## Do not link
+scala/CollectTest.scala
+scala/MatchErrorSerializationTest.scala
+scala/PartialFunctionSerializationTest.scala
+scala/lang/stringinterpol/StringContextTest.scala
+scala/collection/IterableTest.scala
+scala/collection/IteratorTest.scala
+scala/collection/NewBuilderTest.scala
+scala/collection/SeqViewTest.scala
+scala/collection/SetMapConsistencyTest.scala
+scala/collection/SetMapRulesTest.scala
+scala/collection/Sizes.scala
+scala/collection/concurrent/TrieMapTest.scala
+scala/collection/convert/WrapperSerializationTest.scala
+scala/collection/immutable/ChampMapSmokeTest.scala
+scala/collection/immutable/ChampSetSmokeTest.scala
+scala/collection/immutable/LazyListGCTest.scala
+scala/collection/immutable/LazyListLazinessTest.scala
+scala/collection/immutable/ListTest.scala
+scala/collection/immutable/SerializationTest.scala
+scala/collection/immutable/StreamTest.scala
+scala/collection/immutable/StringLikeTest.scala
+scala/collection/immutable/VectorTest.scala
+scala/collection/mutable/AnyRefMapTest.scala
+scala/collection/mutable/ArrayBufferTest.scala
+scala/collection/mutable/OpenHashMapTest.scala
+scala/collection/mutable/PriorityQueueTest.scala
+scala/collection/mutable/SerializationTest.scala
+scala/concurrent/FutureTest.scala
+scala/concurrent/duration/SerializationTest.scala
+scala/concurrent/impl/DefaultPromiseTest.scala
+scala/io/SourceTest.scala
+scala/jdk/AccumulatorTest.scala
+scala/jdk/DurationConvertersTest.scala
+scala/jdk/FunctionConvertersTest.scala
+scala/jdk/OptionConvertersTest.scala
+scala/jdk/StepperConversionTest.scala
+scala/jdk/StepperTest.scala
+scala/jdk/StreamConvertersTest.scala
+scala/jdk/StreamConvertersTypingTest.scala
+scala/math/OrderingTest.scala
+scala/runtime/ScalaRunTimeTest.scala
+scala/sys/env.scala
+scala/sys/process/ParserTest.scala
+scala/sys/process/PipedProcessTest.scala
+scala/sys/process/ProcessBuilderTest.scala
+scala/sys/process/ProcessTest.scala
+scala/tools/testkit/AssertUtilTest.scala
+scala/util/PropertiesTest.scala
+scala/util/SpecVersionTest.scala
+scala/util/SystemPropertiesTest.scala
+
+# Test does not link (uses assertSameElements)
+# scala/scala@1390315b03bb32ffdfb1394b3ec6f4619faca1b8
+scala/collection/mutable/ListBufferTest.scala
```

Comment:
@sjrd not sure what to do about this. Seems like `assertSameElements` hasn't changed in a while, it just hasn't been used.

Reply:
What's the linking error, exactly? I guess we don't have `testkit/AssertUtil.scala` at all. We should probably have it. But otherwise, we can leave it in the "does not link" category.

Reply:
Oh right... The problem is `scala.runtime.ScalaRunTime.stringOf` which needs `getClassLoader` to do some generic (i.e. loader independent) subtyping checks.